src/Uno.UI/UI/Xaml/IFrameworkElement.cs

```diff
@@ -63,6 +63,7 @@ public partial interface IFrameworkElement : IUIElement, IDataContextProvider, I
 		event RoutedEventHandler Unloaded;
 		event EventHandler<object> LayoutUpdated;
 		event SizeChangedEventHandler SizeChanged;
+		//event TypedEventHandler<FrameworkElement, object> ActualThemeChanged;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5709307)